recipes/r-floral/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "FLORAL" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: r-{{ name|lower }}
+  version: '{{ version }}'
+
+source:
+  url: https://cran.r-project.org/src/contrib/FLORAL_{{ version }}.tar.gz
+  sha256: 06993ccb74d5e933412f06e0efa5c43a69e3bde36b3c426279a942e0c405e29d
+
+build:
+  noarch: generic
+  number: 0
+  merge_build_host: true
+  script: R CMD INSTALL --build .
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  run_exports:
+    - {{ pin_subpackage('r-floral', max_pin="x.x") }}
+
+requirements:
+  host:
+    - r-Rcpp >=1.0.9
+    - r-base
+    - r-caret
+    - r-doParallel
+    - r-doRNG
+    - r-dplyr
+    - r-foreach
+    - r-ggplot2
+    - r-glmnet
+    - r-msm
+    - r-mvtnorm
+    - bioconductor-phyloseq
+    - r-reshape
+    - bioconductor-survcomp
+    - r-survival
+  run:
+    - r-Rcpp >=1.0.9
+    - r-base
+    - r-caret
+    - r-doParallel
+    - r-doRNG
+    - r-dplyr
+    - r-foreach
+    - r-ggplot2
+    - r-glmnet
+    - r-msm
+    - r-mvtnorm
+    - bioconductor-phyloseq
+    - r-reshape
+    - bioconductor-survcomp
+    - r-survival
+
+test:
+  commands:
+    - $R -e "library('FLORAL')"
+
+about:
+  home: https://vdblab.github.io/FLORAL/
+  summary: Log-ratio Lasso regression for continuous, binary, and survival outcomes with (longitudinal) compositional features. See Fei and others (2024) <doi:10.1016/j.crmeth.2024.100899>.
+  license: NGPL
```

DESCRIPTION shows `GPL (>=3)`, so

```suggestion
  license: GPL-3.0-or-later
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```